packages/barrels/bin/generate-barrel.js

```diff
@@ -28,7 +29,7 @@ export async function generateBarrels({exclude, directory, cwd}) {
   const directories = (
     await globby(
       directory.map((d) => {
-        return join(d, "*");
+        return joinPosix(d, "*");
```

Comment:
don't all `join` needs to be replaced? i did for my solution, maybe not alias `join` and just use it for all?

Reply:
because for other path like cwd, we have to use the normal path.join function to avoid specific OS issue